documents/operating.md

```diff
@@ -52,14 +52,223 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
 SECTION 5: 🗣🗣🗣 Governing Process 🗣🗣🗣
 ---------------------------------------------------------------------
-- [ ] Types of decision
-- [ ] Records and documentation
-- [ ] Membership
-- [ ] Mechanisms for deliberation
-  - [ ] ... On GitHub
-  - [ ] ... On Slack
-- [ ] Use of emoji
-- [ ] Meetings
+- [x] Types of decision
+- [x] Records and documentation
+- [x] Mechanisms for deliberation
+  - [x] ... On GitHub
+  - [x] ... On Slack
+  - [x] ... Plotting Sessions
+- [x] Use of emoji
+
+### Schedules and Expectations
+All meaningful organizational decisions will happen in monthly Plotting
+Sessions, which are to be scheduled and announced by the Overmind on
+April 1st of every year.  The schedule can be updated as needed, but
+a notice of 10 days must be given before any Plotting Session can be
+held.
+
+April 1st marks the beginning of the governing year, and officer elections
```

Comment:
They're separate -- governing year starts April 1st; Fiscal year will start most likely on January 1st (unless we hear otherwise from lawyers / tax folk)